.github/workflows/screener-build.yml

```diff
@@ -1,20 +1,29 @@
 name: Screener build
 
-on: workflow_dispatch
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  BROWSERSLIST_IGNORE_OLD_DATA: true
+  DEPLOYHOST: 'fluentuipr.z22.web.core.windows.net'
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/186367740-8b5c3ede-67e9-40d4-b0ee-74ebcf10d21c.png)

We have the same on line 65, is it correct?

Reply:
right, I will remove the duplicate